src/inspector_io.cc

```diff
@@ -89,13 +90,18 @@ void HandleSyncCloseCb(uv_handle_t* handle) {
 int CloseAsyncAndLoop(uv_async_t* async) {
   bool is_closed = false;
   async->data = &is_closed;
-  uv_close(reinterpret_cast<uv_handle_t*>(async), HandleSyncCloseCb);
```

Comment:
Great catch! Thank you. I removed it by mistake together with debug statements when debugging the intermittent test failures :/